flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkParquetReader.java

```diff
@@ -98,6 +100,62 @@ public void testTwoLevelList() throws IOException {
     }
   }
 
+  @Test
+  public void testReadBinaryFieldAsString() throws IOException {
+    Schema schemaForWriteBinary = new Schema(optional(1, "strbytes", Types.BinaryType.get()));
+    org.apache.avro.Schema avroSchema = AvroSchemaUtil.convert(schemaForWriteBinary.asStruct());
+
+    File testFile = temp.newFile();
+    Assert.assertTrue(testFile.delete());
```

Comment:
please use AssertJ-style assertions for newly added tests (https://iceberg.apache.org/contribute/#assertj). This makes it easier to migrate from JUnit4 to JUnit5